pypeit/fluxspec.py

```diff
@@ -555,3 +558,390 @@ def __repr__(self):
         txt += '>'
         return txt
 
+
+class EchFluxSpec(masterframe.MasterFrame):
```

Comment:
It is unfortunate that we need an entirely separate class
but I'll admit I don't see an easy way to merge the two.

Reply:
I also didn't find a good way to merge these two classes. It's easy to be called by the flux script to make is a master class.